pkg/controller/servicebindingrequest/retriever_test.go

```diff
@@ -0,0 +1,101 @@
+package servicebindingrequest
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	"github.com/redhat-developer/service-binding-operator/test/mocks"
+)
+
+var retriever *Retriever
+
+func TestRetrieverNew(t *testing.T) {
+	ns := "testing"
+	crdName := "db-testing"
+
+	crdDescription := mocks.CRDDescriptionMock()
+	crd := mocks.DatabaseCRDMock(ns, crdName)
+
+	genericCRDObj, err := runtime.DefaultUnstructuredConverter.ToUnstructured(&crd)
+	assert.Nil(t, err)
```

Comment:
If this is an absolute requirement to proceed further, you can use `require` package: `github.com/stretchr/testify/require`
Same in other places for the initial assertions.

Reply:
Good one! I didn't know about the package, but that's exactly what we need over there. I'm gonna change it.